Makefile

```diff
@@ -5,21 +5,33 @@ build:
 	go build -ldflags "-X 'github.com/conduitio/conduit-connector-postgres.version=${VERSION}'" -o conduit-connector-postgres cmd/connector/main.go
 
 .PHONY: test
-test:
-	# run required docker containers, execute integration tests, stop containers after tests
-	docker compose -f test/docker-compose.yml up --quiet-pull -d --wait
-	go test $(GOTEST_FLAGS) -race ./...; ret=$$?; \
-		docker compose -f test/docker-compose.yml down; \
-		exit $$ret
+test: start_test_service tests stop_test_service 
```

Comment:
This will keep the docker containers running if tests fail 😢

Reply:
Yea, this is the downside, however running `make tests` each time you need a test checked was a little bit much.
Since nearly all tests require running Postgres. 

In all honesty, the makefile have no business managing docker compose.. rather it should just bail if the URL is not provided. 

If we do want to make this easier to clean up - perhaps we can shove it in `t.Main` and add a cleanup hook?

Reply:
The thing is that we use this pattern in all connectors and I prefer to have consistency rather than creating a special way to start/stop containers here. It adds to the cognitive load when working on different repos. So unless it's an important improvement and we want to port it to other connectors as well, I'd rather leave it as it was.

Reply:
can revert it no problem.